### PR TITLE
Add docs for vulnerability scanning

### PR DESCRIPTION
**Pull request by usha-mandya**
Add docs for Vulnerability scanning for Docker local images

### CODE REVIEW

**Review by glours — APPROVED** [review]
LGTM

File: engine/scan/index.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+title: Vulnerability scanning for Docker local images
+description: Vulnerability scanning for Docker local images
+keywords: Docker, scan, Snyk, images, local, CVE, vulnerability, security
+toc_min: 1
+toc_max: 2
+---
+
+## Overview
+
+Vulnerability scanning for Docker local images runs on Snyk engine, providing users with visibility into the security posture of their local Dockerfiles and local images. Users trigger vulnerability scans through the CLI, and use the CLI to view the scan results.  The scanning function creates the list of Common Vulnerabilities and Exposures (CVEs), and provides recommendations for CVE remediations
```
Comment (silvin-lubecki) [inline engine/scan/index.md:11]: nit: missing end point?

File: engine/scan/index.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+1. Download and install Docker Desktop Edge version 2.3.6.0 or later.
+
+    - [Download for Mac](https://desktop.docker.com/mac/edge/Docker.dmg){: target="_blank" class="_"}
+    - [Download for Windows](https://desktop.docker.com/win/edge/Docker%20Desktop%20Installer.exe){: target="_blank" class="_"}
+
+2. Sign into [Docker Hub](https://hub.docker.com){: target="_blank" class="_"}.
+
+3. From the Docker Desktop menu, select **Sign in/ Create Docker ID**. Alternatively, open a terminal and run the command `docker login`.
+
+4. (Optional) You can create a [Snyk account](https://dockr.ly/3ePqVcp){: target="_blank" class="_"} for scans, or use the additional monthly  free scans  provided by Snyk with your Docker Hub account.
```
Comment (silvin-lubecki) [inline engine/scan/index.md:31]: nit: extra spaces around `free scans`

**Review by silvin-lubecki — APPROVED** [review]
LGTM

**Review by silvin-lubecki — APPROVED** [review]
Still LGTM 👍

**Pull request merged**